website/addons/owncloud/static/user-cfg.js

```diff
@@ -0,0 +1,9 @@
+var $osf = require('js/osfHelpers');
+var OwnCloudViewModel = require('./owncloudUserConfig.js').OwnCloudViewModel;
+
+var url = '/api/v1/settings/owncloud/';
+
+var owncloudViewModel = new OwnCloudViewModel(url);
+$osf.applyBindings(owncloudViewModel, '#owncloudAddonScope');
+
+owncloudViewModel.fetch();
```

Comment:
This call should happen when either `OwnCloudUserConfig` or `OwnCloudViewModel` is instantiated, you shouldn't need to call it here.